justfile

```diff
@@ -95,6 +95,9 @@ test-templates:
     just test-template workspace
     just test-template cep78
 
+run-nctl:
```

Comment:
How is this connected to `test-livenet` command?

Reply:
You can run it  to set up nctl before that. It needs to be run once.